src/assets/icons/remix/bin.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24"><path fill="none" d="M0 0h24v24H0z"/><path d="M17 6h5v2h-2v13a1 1 0 0 1-1 1H5a1 1 0 0 1-1-1V8H2V6h5V3a1 1 0 0 1 1-1h8a1 1 0 0 1 1 1v3zm1 2H6v12h12V8zm-9 3h2v6H9v-6zm4 0h2v6h-2v-6zM9 4v2h6V4H9z"/></svg>
```

Comment:
Is this new icon/file being used anywhere?

Reply:
> Should all of these changes of this PR be behind a config flag? Like: Do all CMS users want to allow publishing to multiple contexts? Or do we sometimes want to stick to selecting a single context only? (In the use case of HU for example we'd only want users to select one context.)

IMO yes. I think being able to put content into multiple contexts should be the default and we can add a flag for only being able to put a content into one context at a time.